install.sh

```diff
@@ -216,6 +216,24 @@ parse_args_wrapper() {
     export sudo
 }
 
+check_for_debian_dependencies(){
+    dependenciesNeeded=""
+    if [ "$(which gpg)" == "" ] && [ "$(which gpg2)" == "" ]; then
+        dependenciesNeeded="gpg or gpg2"
+    fi
+    if [ "$(which dirmngr)" == "" ]; then
+        if [ -z "dependenciesNeeded" ]; then
+            dependenciesNeeded="dirmngr"
+        else
+            dependenciesNeeded="$dependenciesNeeded dirmngr"
+        fi
+    fi
+    if [ ! -z "$dependenciesNeeded" ]; then
+        printf "You must have %s installed to add the SignalFx repo keys\n" "$dependenciesNeeded"
```

Comment:
This would say "You must have gpg or gpg2 dirmngr installed to add the SignalFx repo keys" if they didn't have both?  The "or" is ambiguous as to whether you need dirmngr with gpg2 or not and also it isn't a proper sentence.